getting-started/adding-telerik-ui-to-an-application/installation-adding-application-namespace-declaration.md

```diff
@@ -5,60 +5,93 @@ description: Namespace Declaration
 slug: installation-adding-application-namespace-declaration
 tags: namespace,declaration
 published: True
-position: 2
-site_name: Silverlight
+position: 3
+site_name: WPF
 ---
 
 # Namespace Declaration
 
 The following topic will show you how to manually declare a namespace in XAML.
 
-Almost all Telerik controls can be found in a special URI namespace. You can use only this namespace to access all controls from Telerik assemblies that you are referring in your application. You cannot access controls from assemblies that you are not referring.
+Almost all Telerik controls can be found in a special URI namespace schema. You can use only this namespace to access all controls from Telerik assemblies that you are referring in your application. You cannot access controls from assemblies that you are not referring.
         
```

Comment:
Maybe we should specify that we're referring to the "telerik" schema and also make it clearer that some controls have their own* namespace and let clients know that this namespace can be found in the respective Getting Started articles.